zipkin-server/pom.xml

```diff
@@ -117,6 +117,13 @@
       <optional>true</optional>
     </dependency>
 
+    <!-- InfluxDB HTTP backend -->
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>zipkin-autoconfigure-storage-influxdb-http</artifactId>
+      <optional>true</optional>
+    </dependency>
```

Comment:
you are missing a META-INF directory in src/main/resources as well a yaml file. you can look at others for this. The META-INF contents will let the server automagically find this